components/Procedure/src/index.scss

```diff
@@ -79,10 +88,44 @@
   outline-style: unset;
 }
 
-.denhaag-procedure {
-  --denhaag-procedure-step-heading-line-height: var(--utrecht-heading-4-line-height);
-}
-
 .denhaag-procedure .denhaag-process-steps__step-heading {
   font-family: var(--utrecht-heading-font-family);
 }
+
+.denhaag-procedure .denhaag-step-marker::before {
+  counter-increment: denhaag-step-marker;
+  content: counter(denhaag-step-marker);
```

Comment:
Quick question because we use the denhaag-step-marker indicator here. If we have multiple blocks on the same page, will the counter reset to 0 or continue counting? Might be worth testing

Reply:
...deze comment over het hoofd gezien, zal het zo nog even lokaal testen @bleddynacato